src/views/AllSurgeAlerts/index.tsx

```diff
@@ -1,9 +1,187 @@
+import { useContext, useMemo, useState } from 'react';
+import { generatePath } from 'react-router-dom';
+import { isDefined, isNotDefined } from '@togglecorp/fujs';
+
+import Page from '#components/Page';
+import Container from '#components/Container';
+import Pager from '#components/Pager';
+import Table from '#components/Table';
+import {
+    createStringColumn,
+    createDateColumn,
+    createLinkColumn,
+} from '#components/Table/ColumnShortcuts';
+import RouteContext from '#contexts/route';
+import useTranslation from '#hooks/useTranslation';
+import { useRequest } from '#utils/restRequest';
+import type { GoApiResponse } from '#utils/restRequest';
+import { resolveToComponent } from '#utils/translation';
+import { numericIdSelector } from '#utils/selectors';
+import { getDuration } from '#utils/common';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+type SurgeResponse = GoApiResponse<'/api/v2/surge_alert/'>;
+type SurgeListItem = NonNullable<SurgeResponse['results']>[number];
+
+const ITEM_PER_PAGE = 15;
+type TableKey = number;
+const today = new Date().getTime();
+
+function getPositionString(alert: SurgeListItem) {
+    if (isNotDefined(alert.molnix_id)) {
+        return alert.message;
+    }
+    return alert.message?.split(',')[0];
+}
+
+function getMolnixKeywords(molnixTags: SurgeListItem['molnix_tags']) {
+    const filtered = molnixTags.filter((tag) => {
+        if (tag.name.startsWith('OP-')) {
+            return false;
+        }
+        return true;
```

Comment:
We can simplify this by returning the boolean conditional directly

```suggestion
        return !tag.name.startsWith('OP-');
```